### DISCUSSION TITLE
Issues with trame.FloatCard in vue3.

### DISCUSSION THREAD

**Discussion by 1176791355** [Kitware/trame#544] [Q&A] [answered]
FloatCard is a great widget to use, but it seems to behave differently in vue3 and vue2. When using vue3, the FloatCard is covered by the toolbar at the top, whereas in vue2 this issue does not exist.
Also, it seems that any setting regarding its position causes the FloatCard not to be draggable.

```
r"""
Installation requirements:
    pip install trame trame-vuetify trame-components
"""

from trame.app import get_server
from trame.ui.vuetify3 import SinglePageWithDrawerLayout
from trame.widgets import trame

server = get_server()

with SinglePageWithDrawerLayout(server) as layout:
    layout.title.set_text("FloatCard Demo")

    with layout.content:
        trame.FloatCard(
            "Drag the handle to move me anywhere",
            classes="pa-8",
        )

if __name__ == "__main__":
    server.start()
```

**Comment by jourdain** [comment]
```python
with layout.content as content:
   content.style = "position: relative"
   ...
```

**Reply by 1176791355** [reply]
Thanks for the reply, but sorry I didn't follow you.
I'll further express my main confusion through the screenshots of the web ui.
When using the FloatCard example code (vue2), the floatcard hovers over all components normally.
```
from trame.app import get_server
from trame.ui.vuetify import SinglePageLayout
from trame.widgets import trame

server = get_server(client_type="vue2")

with SinglePageLayout(server) as layout:
    layout.title.set_text("FloatCard Demo")

    with layout.content:
        trame.FloatCard(
            "Drag the handle to move me anywhere",
            classes="pa-8",
        )

if __name__ == "__main__":
    server.start()
```
![image](https://github.com/Kitware/trame/assets/93312602/7fd02f3d-930e-49c8-a273-c03168e74b75)

And when switching to vue3, the floatcard is hidden behind the top toolbar.
```
from trame.app import get_server
from trame.ui.vuetify3 import SinglePageLayout
from trame.widgets import trame

server = get_server()
```
![image](https://github.com/Kitware/trame/assets/93312602/a2034da9-6bf3-401c-971a-3eaa9ffa4c54)

**Reply by jourdain** [reply, marked as answer]
The reason is that Vuetify3 is changing how some styles are applied. But you can override it to make it work. Also, you can use [v-drag](https://www.npmjs.com/package/v-drag) with  your own components.

**Reply by 1176791355** [reply]
OK, got it.
